cdk/salmon_cdk_cloudformation_exec_policy.yaml

```diff
@@ -11,22 +11,52 @@ Resources:
             Action:
               - cloudformation:*    # For Nested Stacks (InfraToolingMainStack)
               - ec2:*               # ToolingGrafana
-              - emr-serverless:*    # ToolingAlerting, Monitored
               - events:*            # ToolingAlerting, ToolingMonitoring, Monitored
-              - glue:*              # Monitored
-              - iam:*               # ToolingAlerting, ToolingCommon, ToolingGrafana, ToolingMonitoring, Monitored
               - kms:*               # ToolingCommon, ToolingGrafana, ToolingMonitoring
               - lambda:*            # ToolingAlerting, ToolingCommon, ToolingMonitoring, Monitored
               - logs:*              # ToolingAlerting, ToolingCommon, ToolingGrafana, Monitored
               - s3:*                # ToolingAlerting, ToolingCommon, ToolingGrafana, ToolingMonitoring
-              - secretsmanager:*    # ToolingCommon, ToolingGrafana
               - ses:*               # ToolingCommon
               - sns:*               # ToolingAlerting, ToolingCommon, ToolingMonitoring
               - ssm:*               # Required by CDK itself (ssm:GetParameters)
               - sqs:*               # ToolingAlerting, ToolingCommon, ToolingMonitoring
-              - states:*            # Monitored
               - timestream:*        # ToolingCommon, ToolingGrafana, ToolingMonitoring
             Resource: "*"
+          - Effect: Allow
+            Action:                 # ToolingAlerting, ToolingCommon, ToolingGrafana, ToolingMonitoring, Monitored  
+              - iam:AttachRolePolicy             
```

Comment:
just for the perfectionism - maybe we can rearrange actions.
Like first role-related, then role-attach/detach related, then policy-related etc.

Not critical, of course :)